src/EditorFeatures/CSharpTest/QuickInfo/SemanticQuickInfoSourceTests.cs

```diff
@@ -4181,6 +4181,36 @@ public class Methods2
             await VerifyWithReferenceWorkerAsync(markup, MainDescription(expectedDescription));
         }
 
+        [Fact, Trait(Traits.Feature, Traits.Features.Completion)]
```

Comment:
I see that many of the other tests in this file also have `Completion` here, but it should have `QuickInfo`, as the test below it does.

Also, can you add a `WorkItem` attribute linking it to the reported issue?